CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+  	picture = "mirko_tremaglia"
+	
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1989
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = populist_dictatorship 
+		is_vassal = no
+		war = no
+		
+		ruling_party_ideology = nationalist
+		
+		NOT = { has_country_modifier = dirigisme }
+		has_country_modifier = dirty_war		
+		
+		NOT = { has_country_flag = ITA_Almirante_Renewal }
+		has_country_flag = ITA_Tremaglia
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+	
+	mean_time_to_happen = { months = 1 }
+	
+	option = {
+		name = "Democratization is halted, Tremaglia is ousted"
+		badboy = 3
+		
+		prestige = -50
+		
+		government = nationalist_dictatorship
+		
+		ruling_party_ideology = nationalist
+		
+		#Leadership
+		remove_country_modifier = expert_diplomat
+		remove_country_modifier = successful_industrialist
+		remove_country_modifier = ivory_tower_intellectual
+		remove_country_modifier = man_of_the_people
+		remove_country_modifier = landed_gentleman
+		remove_country_modifier = military_man
+		remove_country_modifier = lord_admiral
+		remove_country_modifier = raving_loon
+		remove_country_modifier = luddite
+		remove_country_modifier = great_statesman
+		remove_country_modifier = backroom_boy
+		remove_country_modifier = great_reformer
+		remove_country_modifier = gauche_buffoon
+		remove_country_modifier = administrative_genius
+		remove_country_modifier = visionary_thinker
+		remove_country_modifier = competent_placeholder
+		remove_country_modifier = imperious_autocrat
+		remove_country_modifier = prince_of_terror
+		remove_country_modifier = efficient_sociopath
+		remove_country_modifier = pig_headed_isolationist
+		remove_country_modifier = ambitious_union_boss
+		remove_country_modifier = dour_economist
+		
+		add_country_modifier = { name = military_man duration = -1 }
+		
+		any_pop = {  
+			consciousness = 4
+			militancy = 4
+		}
+		
+		random_country = {
+			limit = { tag = USA OR = { government = democracy government = democracy1 } is_greater_power = yes }
+			diplomatic_influence  = { who = THIS value = -20 }
+			relation = { who = THIS value = -50 }
+        }
+        
+		clr_country_flag = ITA_Tremaglia
+		set_country_flag = ITA_Miceli
+	}
+	
+}
+
+#Renewal - Good Ending
+country_event = {
+  	id = 8020449
+  	title = "Prepare for New Elections"
+  	desc = EVT_8020449_DESC
+  	picture = "mirko_tremaglia"
+  
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1991
+		month = 1
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = populist_dictatorship 
+		is_vassal = no
+		war = no
+		
+		ruling_party_ideology = nationalist
+		
+		NOT = { has_country_modifier = yugoslaveia_economic_boom }
+		has_country_modifier = dirigisme
+		has_country_modifier = dirty_war
+		
+		has_country_flag = ITA_Tremaglia
+		has_country_flag = ITA_Almirante_Renewal
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+	
+	mean_time_to_happen = { days = 1 }
+	
+	immediate = { 
```

Comment:
These effects should be put in the event option instead. Having too many effects in the immediate bracket will clutter the localisation in the event and make the words spill out of the box.